src/M6Web/Component/RedisMock/RedisMockFactory.php

```diff
@@ -322,6 +325,12 @@ protected function getMethodSignature(\ReflectionMethod $method)
                 } else {
                     $signature .= var_export($parameter->getDefaultValue(), true);
                 }
+            } else if ($parameter->isOptional() && !$variadic) {
+                $signature .= ' = null';
+            }
+
+            if ($variadic) {
+                $signature = '...' . $signature;
```

Comment:
Not sure that it will work with a type variadic argument like:
```php
foo(string ...$bar) {}
```
Looks like it will return `...string $bar`, isn't it?